assets/js/Sign.jsx

```diff
@@ -138,7 +139,7 @@ class Sign extends Component {
                   Off
                 </option>
                 <option value="static_text">
-                  Text
+                  Custom
```

Comment:
Can "Off" go at the bottom of the list too? Currently they show up in alphabetical order

Reply:
my preferred order would be "Auto", "Custom", "Off"

Reply:
Sure, I changed it to that order.

Reply:
<img width="149" alt="screen shot 2019-01-31 at 12 45 45 pm" src="https://user-images.githubusercontent.com/384428/52077246-2930f400-2556-11e9-9591-515b648c7055.png">